src/EA.Weee.Web/Areas/Admin/Views/ProducerSubmission/RemoveSubmission.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@{
+    var title = "Add payment details";
```

Comment:
Not important, but might be worth changing the title here in case testers complain